mock-recipe-server/generate.py

```diff
@@ -22,11 +21,9 @@
 
 
 # Now that Django is set up we can import Django things.
-from normandy.base.utils import canonical_json_dumps  # noqa
-from normandy.recipes.api.serializers import ClientSerializer  # noqa
-from normandy.recipes.models import Action  # noqa
+from django.template import Context, Template  # noqa
 
-from fixtures import get_fixtures  # noqa
+from testcases import get_testcases  # noqa
```

Comment:
The linter complains about imports that aren't at the top of the file, but we can't import anything that relies on Django until after setting up the path stuff. Flake8 sucks are configuring exceptions, so the only good solution I could find was `noqa`, but it's annoyingly broad.